core/templates/pages/volunteer-page/volunteer-page.component.ts

```diff
@@ -114,20 +121,38 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
         },
         {
           content: (
-            '“Oppia turns my drawings into little seeds' +
-            ' for the better world.“'),
-          name: '- Tuna R. Terzi',
-          type: 'Artist'
+            '“I joined Oppia driven by a deep passion for education. ' +
+            'With over five years of experience in edtech, I am committed to ' +
+            'leveraging technology to enhance and enrich the learning ' +
+            'experience, making education accessible and engaging for all.”'
+          ),
+          name: '- liwei zhang',
```

Comment:
Capitalize: Liwei Zhang

Also, the "- " should not be part of the name. It should be part of the HTML. Please drop it from the data here and only include it in the HTML code since it's more of a styling thing and not part of the actual data.

Reply:
Done